### PR TITLE
Fix Rostest issue with waypoint_updater

### LINKED ISSUE #91: ROStest issue: waypoint_updater
I see this in Travis log:

```
[WARN] [1509631658.379411]: WaypointUpdater-Published speed: 10
Traceback (most recent call last):
  File "/capstone/ros/src/waypoint_updater/waypoint_updater.py", line 231, in <module>
    WaypointUpdater()
  File "/capstone/ros/src/waypoint_updater/waypoint_updater.py", line 59, in __init__
    self.publish_waypoints_ahead()
  File "/capstone/ros/src/waypoint_updater/waypoint_updater.py", line 223, in publish_waypoints_ahead
    self.set_waypoint_velocity(lane.waypoints, i, speeds)
  File "/capstone/ros/src/waypoint_updater/waypoint_updater.py", line 105, in set_waypoint_velocity
    waypoints[waypoint].twist.twist.linear.x = velocity
IndexError: list index out of range
testtest_waypoint_updater ... ERROR!
ERROR: max time [10.0s] allotted for test [test_waypoint_updater] of type [waypoint_updater/test_waypoint_updater.py]
```

### PR DESCRIPTION
**Pull request by avrabe**
Fixes #91

Signed-off-by: Ralf Anton Beier <ralf_beier@me.com>

### CODE REVIEW

**Comment by asimonov** [comment]
thanks!
I just merede tl_detector PR.
let's see if (https://travis-ci.org/Kairos-Automotive/carla-brain/builds/296436014#L826) errors clear

**Comment by asimonov** [comment]
can I trigger Travis build again on this PR?

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 1]
```diff
@@ -219,7 +219,8 @@ def publish_waypoints_ahead(self):
 
         rospy.logwarn("WaypointUpdater-Published speed: {}".format(speeds))
```
Comment (asimonov) [inline ros/src/waypoint_updater/waypoint_updater.py]: oh, by the way, can you also change "WaypointUpdater-Published speed" to "wp_updater: published speed"?

**Review by asimonov — APPROVED** [review]
lgtm

File: ros/src/waypoint_updater/waypoint_updater.py (hunk 1 above)
Comment (avrabe) [reply ros/src/waypoint_updater/waypoint_updater.py]: done.

**Review by asimonov — APPROVED** [review]
lgtm

**Review by dabavil — APPROVED** [review]

**Review by vguerra — APPROVED** [review]
lgtm

**Review by vguerra — APPROVED** [review]

**Pull request merged**